src/pages-conditional/eth.tsx

```diff
@@ -16,153 +25,233 @@ import InfoBanner from "../components/InfoBanner"
-`
+const TwoColumnContent = (props: FlexProps) => (
+  <Flex
+    w="full"
+    direction={{ base: "column", lg: "row" }}
+    justify="space-between"
+    p={8}
+    mb={12}
+    {...props}
+  />
+)
 
-const StyledCardContainer = styled(CardContainer)`
-  margin-bottom: 2rem;
-`
+export const LeftColumn = (props: ChildOnlyProp) => (
+  <Box
+    flex="0 0 50%"
+    maxW={{ base: "full", lg: "75%" }}
+    mr={{ lg: 16 }}
+    {...props}
+  />
+)
 
-const TokenCard = styled(HorizontalCard)`
-  min-width: 100%;
-  margin: 0.5rem 0rem;
-  border-radius: 0px;
-`
+export const RightColumn = (props: ChildOnlyProp) => (
+  <Flex
+    flex="0 1 50%"
+    direction="column"
+    justify="center"
+    maxW={{ base: "full", lg: "75%" }}
+    mt={{ base: 12, lg: 0 }}
+    {...props}
+  />
+)
 
-const TextDivider = styled.div`
-  margin-bottom: 2rem;
-  margin-top: 2rem;
-  width: 10%;
-  height: 1px;
-  background-color: ${(props) => props.theme.colors.searchResultBackground};
-  @media (min-width: ${(props) => props.theme.breakpoints.l}) {
-    align-self: flex-start;
-  }
-`
+const SubtitleTwo = (props: ChildOnlyProp) => (
+  <Text fontSize="xl" lineHeight={1.4} color="text300" mb={8} {...props} />
+)
 
-const CentralColumn = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  max-width: 960px;
-  margin: 4rem auto;
-`
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Flex
+    align="center"
+    justify="space-between"
+    direction={{ base: "column-reverse", md: "row" }}
+    {...props}
+  />
+)
 
-const CentralActionCard = styled(ActionCard)`
-  flex: none;
-  margin: 2rem 0;
-  .action-card-image-wrapper {
-    padding: 1rem;
-  }
-  @media (min-width: ${(props) => props.theme.breakpoints.s}) {
-    display: flex;
+const Hero = (props: ChildOnlyProp) => (
+  <Box
+    flex="1 1 100%"
+    maxW="800px"
+    bgSize="cover"
+    bgRepeat="no-repeat"
+    ml={{ base: 0, md: 8 }}
+    {...props}
+  />
+)
 
-    .action-card-image-wrapper {
-      min-width: 260px;
-    }
-    .action-card-content {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-      margin-left: 1rem;
-
-      p {
-        margin-bottom: 0;
-      }
-    }
-  }
-`
+const Header = (props: ChildOnlyProp) => (
+  <Box
+    as="header"
+    flex="1 1 50%"
+    minW="300px"
+    mt={{ base: 6, md: 32 }}
+    {...props}
+  />
+)
+
+const H2 = (prop: HeadingProps) => (
+  <Heading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    mt={0}
+    {...prop}
+  />
+)
+
+const H3 = (props: HeadingProps) => (
+  <Heading
+    as="h3"
+    fontSize={{ base: "xl", m: "2xl" }}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const H4 = (props: HeadingProps) => (
+  <Heading
+    as="h4"
+    fontSize={{ base: "md", md: "xl" }}
+    fontWeight={600}
+    lineHeight={1.4}
+    {...props}
+  />
+)
+
+const CardContainer = (props: FlexProps) => (
+  <Flex wrap="wrap" ml={-4} mr={-4} {...props} />
```

Comment:
```suggestion
  <Flex wrap="wrap" my={-4} {...props} />
```

Reply:
@wackerow shouldn't have been `mx` instead? or why `my`?

Reply:
Oops, yes... should be `mx`